kubernetes/README.adoc

```diff
@@ -0,0 +1,56 @@
+= Kubernetes and Openshift tutorial
+
+This tutorial contains an instruction how to run Infinispan library mode (as a microservice) in Kubernetes/OpenShift.
```

Comment:
|  as a microservice

Not necessarily, it's up to user to label what his app will be :smile: